components/TextFields/src/MDCTextInputController.h

```diff
@@ -129,34 +129,42 @@
 
  The underline is an overlay that can be hidden depending on the editing state of the input text.
 
- Default is UITextFieldViewModeAlways.
+ Default is underlineViewModeDefault.
  */
 @property(nonatomic, assign) UITextFieldViewMode underlineViewMode;
 
 /**
- Default value for above property.
+ Default value for underlineViewMode.
 
- Class property. Replaces UIAppearance for classes that do not inherit from UIView.
+ Default is UITextFieldViewModeAlways.
  */
 @property(class, nonatomic, assign) UITextFieldViewMode underlineViewModeDefault;
 
-/** Color the underline changes to when the input is editing. */
+/** 
+ Color the underline changes to when the input is editing. 
+ 
+ Default is underlineColorActiveDefault.
+ */
 @property(nonatomic, null_resettable, strong) UIColor *underlineColorActive;
 
 /**
- Default value for above property.
+ Default value for underlineColorActive.
 
- Class property. Replaces UIAppearance for classes that do not inherit from UIView.
+ Default is primary color or blue.
```

Comment:
Ha. It's GOING to be primary color when we have support for that. I won't write documentation that takes place in the future.